src/Catty/DataModel/Scene/Scene.swift

```diff
@@ -241,6 +241,20 @@
         return nil
     }
 
+    @objc(copyObjects:)
+    func copyObjects(_ objects: [SpriteObject]) -> [SpriteObject] {
+        var objectsList = [SpriteObject]()
+        for object in objects {
+            guard let copiedObject = self.copy(object, withNameForCopiedObject: object.name) else {
+                NSLog("Error: Copied object is empty!")
+                return objectsList
+            }
+            objectsList.append(copiedObject)
+        }
+        project?.saveToDisk(withNotification: true)
```

Comment:
Is saving the project really necessary? As far as I can see the `self.copy `method is already saving the project. Not ideally, since the project is getting saved again for every object to copy, but okay for the moment.